book/architecture.md

```diff
@@ -1,25 +1,72 @@
 # Architecture
 
-## Prerequisites
+## System Overview
 
-Every OP Stack rollup is composed of four main components.
+OP Succinct enhances OP Stack rollups by adding zero-knowledge proof capabilities while maintaining compatibility with existing OP Stack components. This document explains both the standard OP Stack components and the additional OP Succinct components that enable proving blocks.
```

Comment:
OP Succinct transforms any OP Stack chain into a fully validity-proven rollup with a lightweight upgrade that maintains compatibility with existing OP Stack components.

This document explains the standard OP Stack components and the lightweight modifications OP Succinct adds to enable proving blocks with zero-knowledge proofs using [SP1](https://docs.succinct.xyz/docs/introduction).